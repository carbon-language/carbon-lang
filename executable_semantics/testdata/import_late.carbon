// Part of the Carbon Language project, under the Apache License v2.0 with LLVM
// Exceptions. See /LICENSE for license information.
// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
//
// RUN: ../executable_semantics %s | \
// RUN:   FileCheck %s

package ExecutableSemanticsTest api;

fn main() -> i32 {
  return 0;
}

import ExecutableSemanticsTest library "Nonexistent";

// AUTOUPDATE: ../executable_semantics %s
// CHECK: COMPILATION ERROR: executable_semantics/testdata/import_late.carbon:14: syntax error, unexpected IMPORT, expecting END_OF_FILE
