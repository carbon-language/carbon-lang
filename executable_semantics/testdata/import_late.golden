COMPILATION ERROR: executable_semantics/testdata/import_late.carbon:11: syntax error, unexpected IMPORT, expecting END_OF_FILE
EXIT CODE: 255
